docs/docs/resp3.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Guide for Migrating from RESP2 to RESP3 Replies"
+linkTitle: "Migration guide from RESP2 to RESP3"
+weight: 6
+description: JSON RESP2 to RESP3 replies reference for clients developers
+---
+
+In RESP3, the default value of the optional path argument was changed from `.` to `$`. 
+Due to this change, the replies of some commands have slightly changed. 
+This page provides a brief comparison between RESP2 and RESP3 responses for JSON commands to help developers in migrating their clients from RESP2 to RESP3.
+
+### JSON Command Replies Comparison 
+
+The types are described using a [“TypeScript-like” syntax](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html). `Array<a>` denotes an [array](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#arrays) where the type of elements is known, but the number of elements is not.
```

Comment:
Can we show a couple of before/after query examples?

Reply:
I've updated the table and added example for the `JSON.GET`. All other commands seem to have minor response changes like BulkString -> number.